compute/resource_live_table.go

```diff
@@ -0,0 +1,221 @@
+package compute
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+)
+
+// Live Tables Models
+
+type PipelinesAwsAttributes struct {
+	ZoneId             string `json:"zone_id,omitempty" tf:"optional"`
+	InstanceProfileArn string `json:"instance_profile_arn,omitempty" tf:"optional"`
+}
+
+type PipelinesDbfsStorageInfo struct {
+	Destination string `json:"destination,omitempty" tf:"optional"`
+}
+
+type PipelinesLocalFileInfo struct {
+	Destination string `json:"destination,omitempty" tf:"optional"`
+}
+
+type PipelinesS3StorageInfo struct {
+	Destination      string `json:"destination,omitempty" tf:"optional"`
+	Region           string `json:"region,omitempty" tf:"optional"`
+	Endpoint         string `json:"endpoint,omitempty" tf:"optional"`
+	EnableEncryption bool   `json:"enable_encryption,omitempty" tf:"optional,default:false"`
+	EncryptionType   string `json:"encryption_type,omitempty" tf:"optional"`
+	KmsKey           string `json:"kms_key,omitempty" tf:"optional"`
+	CannedAcl        string `jsono:"canned_acl,omitempty" tf:"optional"`
+}
+
+type PipelinesClusterLogConf struct {
+	Dbfs *PipelinesDbfsStorageInfo `json:"dbfs,omitempty" tf:"optional"`
+	S3   *PipelinesS3StorageInfo   `json:"s3,omitempty" tf:"optional"`
+}
+
+type PipelinesInitScriptInfo struct {
+	Dbfs *PipelinesDbfsStorageInfo `json:"dbfs,omitempty" tf:"optional"`
+	S3   *PipelinesS3StorageInfo   `json:"s3,omitempty" tf:"optional"`
+	File *PipelinesLocalFileInfo   `json:"file,omitempty" tf:"optional"`
+}
+
+type PipelinesClusterAttributes struct {
+	SparkConf        map[string]string        `json:"spark_conf,omitempty"`
+	AwsAttributes    *PipelinesAwsAttributes  `json:"aws_attributes,omitempty"`
+	NodeTypeId       string                   `json:"node_type_id,omitempty"`
+	DriverNodeTypeId string                   `json:"driver_node_type_id,omitempty"`
+	SshPublicKeys    string                   `json:"ssh_public_keys,omitempty"`
+	CustomTags       map[string]string        `json:"custom_tags,omitempty"`
+	ClusterLogConf   *PipelinesClusterLogConf `json:"cluster_log_conf,omitempty"`
+	SparkEnvVars     map[string]string        `json:"spark_env_vars,omitempty"`
+	InstancePoolId   string                   `json:"instance_pool_id,omitempty"`
+}
+
+type PipelinesAutoScale struct {
+	MinWorkers int `json:"min_workers"`
+	MaxWorkers int `json:"max_workers"`
+}
+
+type PipelinesClusterSize struct {
+	NumWorkers int                 `json:"num_workers,omitempty"`
+	Autoscale  *PipelinesAutoScale `json:"autoscale,omitempty"`
+}
+
+type PipelinesNewCluster struct {
+	PipelinesClusterAttributes
+	PipelinesClusterSize
+}
+
+type PipelineCluster struct {
+	Label string `json:"label,omitempty"`
+	PipelinesNewCluster
```

Comment:
why do we need to create many types without extra methods, if they are used only in one place?...